cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,44 @@
-describe('', () => {
-  before(() => {
+describe('Article flow', () => {
+  let user;
+  let article;
 
-  });
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
 
-  it('', () => {
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
 
   });
 
-  it('', () => {
+  it('should create an article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.visit('/editor');
+    cy.findByPlaceholder('Article Title').type(article.title);
+    cy.findByPlaceholder('What\'s this article about?')
+    .type(article.description);
+    cy.findByPlaceholder('Write your article (in markdown)').type(article.body);
+    cy.findByPlaceholder('Enter tags').type(article.tags);
```

Comment:
it always fails on that step
![image](https://github.com/mate-academy/qa_cypress_e2e_article_flow/assets/85580286/95741b18-8913-434b-ab0c-488159398159)

Reply:
fixed